godot-core/src/log.rs

```diff
@@ -119,3 +133,26 @@ pub fn print(varargs: &[Variant]) {
     // TODO use generated method, but figure out how print() with zero args can be called
     // crate::engine::utilities::print(head, rest);
 }
+
+/// Prints to the Godot console, used by the [`godot_print_rich!`] macro.
+pub fn print_rich(varargs: &[Variant]) {
```

Comment:
This should probably be `#[doc(hidden)]`, until we know whether we want to use this, or the utility function, or both.

You can also add `#[doc(hidden)]` to `print` above.